src/stepfunctions/steps/service.py

```diff
@@ -35,7 +67,13 @@ def __init__(self, state_id, **kwargs):
             result_path (str, optional): Path specifying the raw input’s combination with or replacement by the state’s result. (default: '$')
             output_path (str, optional): Path applied to the state’s output after the application of `result_path`, producing the effective output which serves as the raw input for the next state. (default: '$')
         """
-        kwargs[Field.Resource.value] = 'arn:aws:states:::dynamodb:getItem'
+
+        """
+        Example resource arn: arn:aws:states:::dynamodb:getItem
+        """
+
+        kwargs[Field.Resource.value] = get_service_integration_arn(DYNAMODB_SERVICE_NAME,
+                                                                   DynamoDBApi.GetItem)
```

Comment:
thanks for adding these through the PR, it definitely helps as a review and will benefit contributors!